lib/miq_automation_engine/models/miq_ae_yaml_import.rb

```diff
@@ -8,6 +8,10 @@ def initialize(domain, options)
     @domain_name = domain
     @options     = options
     @restore     = @options.fetch('restore', false)
+    tenant_id    = @options.fetch('tenant_id', Tenant.root_tenant.try(:id))
```

Comment:
I'm guessing that there will always be a root tenant, so I don't think the raise is necessary.
Also, don't inject root tenant id, to then have us look it up by id.

Think you want these 4 lines to be:

```ruby
@tenant = @options.fetch('tenant') { Tenant.root_tenant } # or maybe it needs to be the second parameter
```